views/partials/addReview.ejs

```diff
@@ -1,18 +1,18 @@
 <script src="/js/addReviewClient.js" defer></script>
 
 <div class="add-review">
-    <button class="add-review-button">Write a review</button>
+    <button class="add-review-button">Skriv en recension</button>
     <div class="add-review-content">
       <form>
-        <label for="name">Name:</label>
+        <label for="name">Namn:</label>
         <input type="text" id="name" name="name" /><br />
 
-        <label for="rating">Rating:</label>
-        <input type="number" min="0" max="5" id="rating" name="rating" /><br />
+        <label for="rating">Betyg:</label>
+        <input type="number" id="rating" name="rating" /><br />
```

Comment:
Tog du bort min="0" max="5" för att det skulle bli lättare att testa felhantering "checkRating()"?

Reply:
I am planning on removing the current input field anyway, so there was no need to keep the min and max, it will already be 5 hats to click, no more, no less :)